config/v12/typo3-120.php

```diff
@@ -25,4 +25,5 @@
     $rectorConfig->rule(\Ssch\TYPO3Rector\Rector\v12\v0\typo3\ReplaceTSFECheckEnableFieldsRector::class);
     $rectorConfig->rule(ReplaceContentObjectRendererGetMailToWithEmailLinkBuilderRector::class);
     $rectorConfig->rule(RemoveUpdateRootlineDataRector::class);
+    $rectorConfig->rule(\Ssch\TYPO3Rector\Rector\v12\v0\typo3\ReplaceTSFEWithContextMethodsRector::class);
```

Comment:
you can import that namespace to be consistent.

Reply:
I am gonna merge. Shouldn´t be a manually task. Should be done by rector. Will have a look why it is not changed.